executor/set_test.go

```diff
@@ -1408,6 +1408,34 @@ func (s *testSuite5) TestSetClusterConfigJSONData(c *C) {
 	}
 }
 
+func (s *testSuite5) TestIssue28230(c *C) {
```

Comment:
I know there are other exceptions, but it's recommended to name the issue something descriptive and add a comment for the issue. i.e. `TestSQLAutoIsNull()`